### DISCUSSION TITLE
Renovate Updates the wrong lines with gradle and I don't know why

### DISCUSSION THREAD

**Discussion by halbekanne** [renovatebot/renovate#16483] [Request Help] [answered]
Two times now, when Renovate suggested changes for a build.gradle file, it changed the wrong lines and I haven't figured out why that could be.

The first one was "Update mapstruct monorepo to v1.5.2.Final" where it should have updated mapstruct dependencies from 1.4.2.Final to 1.5.2.Final. This was the replacement Renovate made:

```
    implementation 'org.mapstruct:mapstruct:1.4.2.Final'
    annotationProcessor 'org.mapstruct:mapstruct1.5.2.Final:1.4.2.Final'

    implementation group: '<some other group>', n1.5.2.Final: '<some other dependency>'
```

It looks like Renovate got confused about where to put the replacement version. The same happened again, it should have updated the depsVersion variable from 2.45.3 -> 2.52.3 and updated it in this way:

```
project.ext {
    depsVersion = '2.45.3'
    lombokVersion = '1.18.22.52.4'
}
```

This only happened with updating build.gradle files, updating npm dependencies has worked so far. This is running on Renovate version 32.105.0 for Code hosted on Bitbucket Server Version 7.17.

Has anyone experienced something similar and is able to help me out with this problem?

**Comment by StinkyLord** [comment, marked as answer]
do u use windows? 

https://docs.renovatebot.com/getting-started/installing-onboarding/#self-hosting-on-windows

**Comment by halbekanne** [comment]
Ah, the line endings are probably it. I'll try it out on monday and see if this fixes it. Thanks!

**Reply by viceice** [reply]
https://docs.renovatebot.com/getting-started/installing-onboarding/#self-hosting-on-windows
